origin.spec

```diff
@@ -35,6 +35,9 @@
 %global make_redistributable 1
 %endif
 
+# by default build the test binaries for Origin
+%global build_tests 1
+
```

Comment:
(I also was meaning to make this change in such a way that someone who wanted the current default behavior had to change nothing to keep getting it)